packages/features/schedules/components/DateOverrideList.tsx

```diff
@@ -124,6 +127,7 @@ const DateOverrideList = ({
                 StartIcon="trash-2"
                 onClick={() => {
                   replace([...fields.filter((currentItem) => currentItem.id !== item.id)]);
+                  handleAvailabilityUpdate();
```

Comment:
Automatically save in case of deleting an override as well. Not sure if this is needed or not, lmk if this needs to be removed